installer/src/main/bin/compareModel.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# *****************************************************************************
+# compareModel.sh
+#
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+#     NAME
+#       compareModel.sh - WLS Deploy tool to compare two models
+#
+#     DESCRIPTION
+#       This script compares two models. The models compared must be both yaml or both json
+#
+# This script uses the following variables:
+#
+# JAVA_HOME             - The location of the JDK to use.  The caller must set
+#                         this variable to a valid Java 7 (or later) JDK.
+#
+
+
+usage() {
+  echo ""
+  echo "Usage: $1 [-help]"
+  echo "          -oracle_home <oracle_home> "
+  echo "          [-compare_model_output_dir <output_dir> write the outputs to the directory specified]"
```

Comment:
Maybe shorten to -output_dir ? if we do need to reuse that name in the future, cla_utils can deal with it, similar to checks for -domain_home from different tools.

Opinions?

Reply:
Yes, fixed